client/app/private-database/translations/Messages_fr_FR.xml

```diff
@@ -20,6 +20,8 @@
    <translation id="privateDatabase_dashboard_version_postgresql_10" qtlid="444443">PostgreSQL 10</translation>
    <translation id="privateDatabase_dashboard_version_mongodb_34" qtlid="448301">MongoDB 3.4</translation>
    <translation id="privateDatabase_dashboard_version_redis_32" qtlid="448314">Redis 3.2</translation>
+   <translation id="privateDatabase_dashboard_version_mongodb_40">MongoDB 4.0</translation>
```

Comment:
Could you please automate this process ? Having to rework every time a new version is released...

Reply:
How can we automate it? I think we cant automate it. All new versions must be manually added to translation files.